api/src/main/resources/workbench-api.yaml

```diff
@@ -3657,6 +3672,24 @@ definitions:
       defaultCdrVersionId:
         type: string
         description: ID of the CDR versions that should be used by the user by default
+  CdrVersionMapResponse:
+    type: array
+    items:
+      type: object
+      required:
+        - accessTierShortName
+        - defaultCdrVersionId
+        - versions
+      properties:
+        accessTierShortName:
+          type: string
+        defaultCdrVersionId:
+          type: integer
```

Comment:
`cdrVersionId` is type string below and in `CdrVersionListResponse` - why the divergence here?

Reply:
in the DB it's a number .... but yes it's already a string in too many places so I'll revert for consistency